web_server/login.py

```diff
@@ -9,16 +9,12 @@
 def login():
     if request.method == "POST":
         if request.form.get("name") in users:
-            print('user is ok')
             if request.form.get("password") == users[request.form.get("name")]:
```

Comment:
Check here if password correspond to user as you save information related to users.
For example, if you save info as dictionary, you need to access `password`:  `users[request.form.get("name")]["password"]`.